### PR TITLE
Ag 10529

### PR DESCRIPTION
**Pull request by jacobp100**
https://ag-grid.atlassian.net/browse/AG-10621

### CODE REVIEW

File: packages/ag-charts-community/src/module/baseModule.ts [hunk 1]
```diff
@@ -1,4 +1,11 @@
+import type { BBox } from '../integrated-charts-scene';
+import type { DataController } from '../module-support';
```
Comment (alantreadway) [inline packages/ag-charts-community/src/module/baseModule.ts]: Nit: Avoid importing from the root bundles.

File: packages/ag-charts-community/src/chart/navigator/miniChart.ts [hunk 2]
```diff
@@ -0,0 +1,644 @@
+import { toRadians } from '../../integrated-charts-scene';
+import type { ModuleContext } from '../../module-support';
```
Comment (alantreadway) [inline packages/ag-charts-community/src/chart/navigator/miniChart.ts]: Ditto

**Review by alantreadway — APPROVED** [review]
LGTM

File: packages/ag-charts-community/src/chart/navigator/navigator.ts [hunk 3]
```diff
@@ -93,31 +108,56 @@ export class Navigator extends BaseModuleInstance implements ModuleInstance {
...
+            this.y = 0;
         }
 
         return { shrinkRect };
     }
 
-    private layoutComplete({ series: { rect, visible } }: LayoutCompleteEvent) {
+    async performCartesianLayout(opts: { seriesRect: BBox }): Promise<void> {
+        const { x, width } = opts.seriesRect;
+        const visible = true;
```
Comment (iMoses) [inline packages/ag-charts-community/src/chart/navigator/navigator.ts:144]: is this on purpose?

**Pull request merged**